README.md

```diff
@@ -64,87 +79,71 @@ Which asks to create the class:
 
 ![Product created](/docs/images/product-created.png?raw=true)
 
-Running Behat again will result in a similar process for the `namedAndPriced` method.
-Where an example for the method will be added to the spec and the method added
-to the class.
+Running Behat again will result in a similar process for the `namedAndPriced`
+method. Where an example for the method will be added to the spec and the
+method added to the class.
 
 ## How
 
-### Installation
-
-This package provides an extension for PhpSpec and an extension for Behat. Both need
-to be enabled for the full functionality.
-
-
-Requires:
+### Requirements
 
 * Behat 3.0+
 * PhpSpec 3.0+
 * PHP 5.6+
 
-Require the extension:
+**Note!** In order to use `BehatSpec` with PHPSpec 2.0 series and PHP 5.4, use
+`0.3.*` version series and check [install instructions for 0.3.* version][3].
 
-```
-$ composer require --dev rmiller/behat-spec:^0.5
-```
+### Installation
 
-In order to use `BehatSpec` with PHPSpec 2.0 series, use `0.3.*` version series:
+Install this package as a development requirement in your project:
 
-```bash
-$ composer require --dev rmiller/behat-spec:0.3.*
 ```
-
-To get the phpspec run command running, you need to use latest phpspec >2.1.
-Otherwise that functionality will silently fail.
+$ composer require --dev rmiller/behat-spec
+```
 
 ### Configuration
 
-Activate the Behat extension by specifying its class in your `behat.yml`:
+Activate the [Behat][1] extensions in `behat.yml` of your project:
 
 ```yaml
 # behat.yml
 # ...
-extensions:
-    RMiller\BehatSpec\Extension\BehatSpecExtension\BehatExtension:
-        path:  bin/phpspec #default value is bin/phpspec
-        config:  path/to/phpspec.yml #optional
+default:
+    # ...
+    extensions:
+        RMiller\BehatSpec\Extension\BehatSpecExtension\BehatExtension:
+            path:  bin/phpspec #default value is bin/phpspec
+            config:  path/to/phpspec.yml #optional
+        RMiller\BehatSpec\Extension\ErrorExtension\ErrorExtension: ~
```

Comment:
This shouldn't be necessary `RMiller\BehatSpec\Extension\BehatSpecExtension\BehatExtension` registers the other two extensions and passes that config to them.